cpp/models/abm/README.md

```diff
@@ -1,57 +1,147 @@
+
+Another way of migration we use in the simulation of Braunschweig (simulations/abm_braunschweig.cpp) is using trips. A trip consists of the ID of the person that performs this trip, a time point when this trip is performed and where the person is heading to. At the beginning of the simulation, a list with all trips is initialized and followed during the simulation. There can be different trips on the weekend than during the week, but other than that, the agents do the same trips every day. As before, agents that are in quarantine or in the hospital cannot migrate.
+
+## Get Started
+
+This section gives an introduction to how to use the ABM and set up your own simulation. For a quick overview, can find a full example in the [ABM minimal example](../../examples/abm_minimal.cpp) and a more detailed Doxygen documentation [here](https://scicompmod.github.io/memilio/documentation/index.html ). For a guide on installation and running the simulations and examples, see this [README](../../README.md).
+
+Every person in the ABM belongs to an AgeGroup, which we can define as the following. Note that every age group has to have values strictly smaller than num_age_groups.
+
+```cpp
+size_t num_age_groups         = 4;
+const auto age_group_0_to_4   = mio::AgeGroup(0);
+```
+
+The initial empty world is created with the number of age groups:
+
+```cpp
+auto world = mio::abm::World(num_age_groups);
+```
+
+We can set several general parameters, which you can find [here](../abm/parameters.h). Here is an example where we set the duration of the incubation period to 4 days:
+
+```cpp
+world.parameters.get<mio::abm::IncubationPeriod>() = 4.
+```
+
+To add a location to the world, we have to specify the kind of location.
+
+```cpp
+auto home = world.add_location(mio::abm::LocationType::Home);
+```
+
+People are added with an age. Then we have to assign them, so the world knows they can travel to this location.
+
+```cpp
+auto person = world.add_person(home, mio::AgeGroup(0));
+person.set_assigned_location(home);
+```
+
+For adding more people to the world, we create households. A Household holds a vector with HouseholdMembers, i.e. a vector with weighted age distribution from which the age of the Persons belonging to this Household can be calculated. A Household and the number of times it exists is gathered in a HouseholdGroup.
+For example, we have children who either belong to AgeGroup(0) or AgeGroup(1) with probability 0.5 in each case and parents which belong to AgeGroup(2) or AgeGroup(3) similarly. We then add households with a parent and a child and households with two parents and one child.
```

Comment:
This is somewhat hard to read, please be more descriptive, e.g. "For example, to create a two parent household with two children ..." or similar.